ocaml/perftest/perfutil.ml

```diff
@@ -25,7 +25,7 @@ let rpc xml =
 let remoterpc host xml =
   let open Xmlrpc_client in
   XMLRPC_protocol.rpc ~srcstr:"perftest" ~dststr:"remotexapi"
-    ~transport:(SSL (SSL.make (), host, 443))
+    ~transport:(SSL (SSL.make ~verify_cert:None (), host, 443))
```

Comment:
@edwintorok Do we know under which circumstances this connection is opened? Is it always to another host in the pool?

Reply:
It would be acceptable to not use verification for debugging or other non-production code. It would be also safe to set verification if we know for certain the destination.